native/dggsvp3.go

```diff
@@ -260,7 +260,7 @@ func (impl Implementation) Dggsvp3(jobU, jobV, jobQ lapack.GSVDJob, m, p, n int,
 		}
 
 		// Clean up A.
-		for i := k + 1; i < k+l; i++ {
+		for i := k + 1; i < min(m, k+l); i++ {
```

Comment:
The docs promise that it will be zero, so it's nice to keep that promise. We should really have a way to create bad matrices for testing.